tests/unit/test_sql_analyser.py

```diff
@@ -308,6 +308,10 @@ def test_create_index_concurrently_where(self):
         sql = 'CREATE INDEX CONCURRENTLY "index_name" ON "table_name" ("a_column") WHERE ("some_column" IS NOT NULL);'
         self.assertValidSql(sql)
 
+    def test_create_unique_index_concurrently_where(self):
+        sql = 'CREATE UNIQUE INDEX CONCURRENTLY "index_name" ON "table_name" ("a_column") WHERE ("some_column" IS NOT NULL);'
+        self.assertValidSql(sql)
```

Comment:
Adding a unique index is considered backward incompatible 👀 
See https://github.com/3YOURMIND/django-migration-linter/blob/main/docs/incompatibilities.md#arrow_forward-adding-a-unique-constraint

Reply:
good catch, I think tests should be updated now to assert that it's at leas the uniqueness error and not the not null error